core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -332,10 +332,14 @@ class AvroUpsertMonitorActor(val pollInterval: FiniteDuration,
                 )
               }
             case Failure(t) =>
+              val errMsg = t match {
+                case errRpt: RawlsExceptionWithErrorReport => errRpt.errorReport.message
+                case x                                     => x.getMessage
+              }
```

Comment:
this fixes the outermost stringification in the case we have a `RawlsExceptionWithErrorReport`.